kml/serdes.cpp

```diff
@@ -206,7 +167,7 @@ std::string TimestampToString(Timestamp const & timestamp)
   return strTimeStamp;
 }
 
-void SaveLocalizableString(KmlWriter::WriterWrapper & writer, LocalizableString const & str,
+void SaveLocalizableString(Writer & writer, LocalizableString const & str,
```

Comment:
Would it be hard to extract this refactoring and merge it to the master separately? Actually, some other changes where kml-only code is shared between gpx can also be safely merged to master, right?

It will reduce the PR size and simplify the review.

Reply:
Commit with this change is already squashed, so it will be required to manually cherry-pick subsets of lines to separate PR. Not sure it will give many benefits, given this one is close to be ready.

Reply:
For the future: if you see that some change can be done in a separate commit and merged in a separate PR, then do it, to avoid stalling the PR and growing the list of changes to review. It saves a significant amount of time for us.

P.S. It is an important skill to amend the commit and extract only necessary changes to a different commit. There are several convenient GUI git apps that help with commit amendment. Then that different commit can be easily cherry-picked to a different branch to be merged into the master independently.